package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "dynamic-webview-builder",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^6.4.2",
+    "@fortawesome/free-solid-svg-icons": "^6.4.2",
+    "@fortawesome/react-fontawesome": "^0.2.0",
```

Comment:
Why do we have multiple libraries for icons? Can we only pick one, please.

keep in mind that the more modules you add the bigger the bundle size of the project which affects the performance 🙈